Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -288,16 +288,16 @@ def is_investigation_mirrored(investigation_id: str, mirrored_channels: list) ->
 
 def urlify_hyperlinks(message: str, url_header: str | None = EXTERNAL_FORM_URL_DEFAULT_HEADER) -> str:
     """
-    Turns URL to markdown hyper-link
+    Converts URLs to Markdown-format hyperlinks.
     e.g. https://www.demisto.com -> [https://www.demisto.com](https://www.demisto.com)
     :param message: Message to look for URLs in
-    :return: Formatted message with hyper-links
+    :return: Formatted message with hyperlinks.
     """
     url_header = url_header or EXTERNAL_FORM_URL_DEFAULT_HEADER
     formatted_message: str = message
-    # URLify markdown hyperlinks
-    urls = re.findall(URL_REGEX, message)
-    for url in urls:
+
+    for url_match in re.finditer(urlRegex, message):
+        url: str = url_match.group()
```

Comment:
why needed `:str`?

Reply:
The `group()` method returns `AnyStr` and not `str`, while `replace` receives `str`, which seems to confuse IDEs (both VSCode and PyCharm complain)
Not sure if it fails mypy checks though.

<img width="764" alt="Screenshot 2023-07-02 at 14 15 08" src="https://github.com/demisto/content/assets/8832013/ed0299dd-4e75-4aa7-991b-97abdf3383c7">